packages/assets-controllers/src/CurrencyRateController.ts

```diff
@@ -82,11 +83,14 @@ const defaultState = {
  * Controller that passively polls on a set interval for an exchange rate from the current network
  * asset to the user's preferred currency.
  */
-export class CurrencyRateController extends PollingController<
-  typeof name,
-  CurrencyRateState,
-  CurrencyRateMessenger
-> {
+export class CurrencyRateController
+  extends StaticIntervalPollingController<
+    typeof name,
+    CurrencyRateState,
+    CurrencyRateMessenger
+  >
+  implements IPollingController
```

Comment:
It isn't necessary to say that CurrencyRateController implements IPollingController; we already know StaticIntervalPollingController does, because we said it did. So I believe we can drop this line from this controller (and all of the other ones)?

Reply:
yep you're right. That was redundant. Removed here: https://github.com/MetaMask/core/pull/3636/commits/823aaa188f572a8aa76973a012ae9bfa8da518e0